sdk/src/azure/platform/az_curl.c

```diff
@@ -305,8 +305,7 @@ static AZ_NODISCARD az_result _az_http_client_curl_send_get_request(CURL* ref_cu
 /**
  * handles DELETE request
  */
-static AZ_NODISCARD az_result
-_az_http_client_curl_send_delete_request(CURL* ref_curl)
+static AZ_NODISCARD az_result _az_http_client_curl_send_delete_request(CURL* ref_curl)
```

Comment:
nit: It's fine here, but ideally, such formatting changes on files that you would otherwise not be included in a PR should be done separately.

Reply:
It's just whenever I modify a file, before saving, I run auto-format (ctrl+K-D). I think separate PR is fine, but we always should be auto-formatting.

Reply:
I agree, and that makes sense (I do the same). But this PR isn't otherwise modifying `az_curl.c`.